examples/forecasting/Forecasting_Task_Quick_Start.ipynb

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
Can this be deleted since we now have quickstart tutorial in docs/tutorials/?

Reply:
will be removed once docs are revised 👍